pkg/apis/workflow/v1alpha1/generated.proto

```diff
@@ -178,6 +178,17 @@ message CronWorkflow {
   optional CronWorkflowStatus status = 3;
 }
 
+message CronWorkflowCondition {
```

Comment:
can't we reuse the existing condition struct?

Reply:
that way you don't need Jesse approval

Reply:
When I first started this change I decided against it because I wanted to keep the `type` parameters (i.e. `CronWorkflowConditionType`) separate.

However, after finishing the implementation I agree that reusing the existing struct is better. I'll refactor